pkg/config/config.go

```diff
@@ -307,6 +307,7 @@ func init() {
 	Datadog.BindEnv("cluster_agent.cmd_port")
 	Datadog.BindEnv("cluster_agent.kubernetes_service_name")
 	BindEnvAndSetDefault("hpa_watcher_polling_freq", 10)
+	BindEnvAndSetDefault("hpa_configmap_name", "datadog-hpa")
```

Comment:
I think we should change the default name of the `configmap` to something like `datadog-custom-metrics`. What do you think?

Reply:
agreed, I'm adding this in the last item of the feedback card.